src/table-list-field/table-list-field.component.ts

```diff
@@ -63,6 +63,8 @@ export class TableListFieldComponent extends AbstractListFieldComponent implemen
 
   onFieldAdd(field: string) {
     this.keys.push(field);
+    setTimeout(() => {
```

Comment:
No, actually this ensures that the `sortAndSynchronize` method in shortcutActionService will be called after all new elements are created and pushed in the `elemTabIndexList`. In this way, i can sort only one time. Otherwise i must trigger sorting every time a new element is added. That's why i insert only a new empty element in parent array( for example a new item in authors such as authors.1) and all children of this element are created from angular bindings.